source/fundamentals/atlas-search.txt

```diff
@@ -0,0 +1,606 @@
+   covered by an :atlas:`Atlas search index </reference/atlas-search/index-definitions/>`.
+   Learn more about the required setup and the functionality of this operator
+   from the :ref:`Atlas Search <fts-top-ref>` documentation.
+
+The examples in this guide use the following documents in a collection called
+``guitars``:
+
+.. code-block:: json
+ 
+   { "_id": 1, "make": "Fender", "description": "Classic guitars known for their versatility.", "establishedYear": 1946, "in-stock": true, "rating": 9 }
+   { "_id": 2, "make": "Gibson", "description": "Classic guitars known for their rich, full tones.", "establishedYear": 1902, "in-stock": true, "rating": 8 }
+   { "_id": 3, "make": "PRS", "description": "High-end guitars known for their quality.", "establishedYear": 1985, "in-stock": true, "rating": 9 }
+   { "_id": 4, "make": "Kiesel", "description": "Quality guitars made only for custom orders.", "establishedYear": 2015, "in-stock": false }
+   { "_id": 5, "make": "Ibanez", "description": "Well-crafted guitars used by many professional guitarists.", "establishedYear": 1957, "in-stock": true, "rating": 7 }
+   { "_id": 6, "make": "Strandberg", "description": "Modern guitars known for their headless models.", "establishedYear": 1982, "in-stock": false }
+
+The following ``Guitar`` class models the documents in this collection.
+
+.. code-block:: csharp
+
+   public class Guitar
+   {
+       public int Id { get; set; }
+       public string Make { get; set; }
+       public List<string> Models { get; set; }
+       public int EstablishedYear { get; set; }
+       [BsonElement("in-stock")]
+       public bool InStock { get; set; }
+       public int? Rating { get; set; }
+   }
+
+.. note::
+
+   The documents in the ``guitars`` collection use the camel-case naming
+   convention. The examples in this guide use a ``ConventionPack``
+   to deserialize the fields in the collection into Pascal case and map them to
+   the properties in the ``Guitar`` class.
+
+   To learn more about custom serialization, see :ref:`csharp-custom-serialization`.
+
+Create an Atlas Search Index
+----------------------------
+
+Before you can perform a search on an Atlas collection, you must first create an **Atlas
+Search index** on the collection. An Atlas Search index is a data structure that
+categorizes data in a searchable format. 
+
+To learn how to create an Atlas Search Index see the
+:atlas:`Create an Atlas Search Index </atlas-search/create-index>` Atlas guide.
+
+Atlas Search Operators
+----------------------
+
+The ``Search`` class contains methods you can use to perform ``$search``
+operations. For a full list of available ``$search`` operations, see the :atlas:`Operators and Collectors
+</atlas-search/operators-and-collectors>` Atlas guide.
+
+.. note:: 
+
+   The ``Search`` class does not currently support the ``$embeddedDocument`` operator.
+
+Autocomplete
+~~~~~~~~~~~~
+
+Use the ``Autocomplete`` method to search for a word or phrase that contains a
```

Comment:
Grammar

In all of these sections, the method should not be capitalized. Later in the text for each one you don't capitalize it and the linked reference pages also don't. For example:

Use the `autocomplete` method...

I'm not sure if the title should be capitalized because I realize it's a section heading, but the method itself is not capitalized. I'm not sure what the convention is and what takes precedence.

Reply:
The distinction here is that the C# method name is capitalized, but the Atlas operator isn't. So any time it is capitalized it should be followed by "method", and in other cases it should be followed by "operator". I agree this is a little confusing, but I wasn't sure of a better way to go about it with that distinction.

Side note - I did realize I used "operation" instead of "operator" in the links, so I changed those.